### ISSUE TITLE
Automatically writing start commit message summary upon keyboard press

### ISSUE THREAD

**Issue by emmadesilva** [desktop/desktop#15350]
### The feature request

It would be very nice to have it so that when in GitHub Desktop, and you start typing on the keyboard, the commit summary input automatically gets highlighted and the text entered right away.

### Proposed solution

This would save the step of having to move the mouse to the input (or using the shortcut `CTRL+G`) when creating commits.

### Additional context

This shouldn't be triggered when a modifier key is pressed, or an other element has focus.

**Comment by angusdev** [comment]
I don't think it is a good idea. The behaviour is unexpected and unpreditable to user.  Also I don't think commit is a frequent operation that worth to steal the keystrokes of the app.

Or is there any special use case?

**Comment by emmadesilva** [comment]
> I don't think it is a good idea. The behaviour is unexpected and unpreditable to user. Also I don't think commit is a frequent operation that worth to steal the keystrokes of the app.
> 
> Or is there any special use case?

For me personally it would be a frequent operation. On the days I code I on average make around a hundred commits per day using GitHub desktop. Though I realize that is likely an extreme use case.
